src/components/ITSBTooltip/ITSBTooltip.js

```diff
@@ -0,0 +1,20 @@
+export const ITSBTooltip = props => {
+
+  const { object, graph } = props;
+
+  if (object) {
+    const { present } = object;
+
+    const people = Object.entries(present)
+      .map(([ authorId, likelihoods ]) => {
+        const authorName = graph.getNode(authorId).name;
+
+
+        // right now this is just showing likelihoods, for debugging
+        return `${authorName}: ${Array.from(likelihoods).join(', ')}`;
+      }).join('\n');
+      
+    return `${object.properties.title}\n\n${people}`;
```

Comment:
For some reason this tooltip isn't showing up for me on either visualization. Not a big deal because it was only ever intended to be for temporary debugging anyway; eventually, some info about each intersection needs to go into a sidebar that shows up when you click on one of the intersections in the visualization.

Reply:
Ok, interesting. I just tried and it definitely works for me in the "intersections" mode, both in Chrome and FF. (The arcs aren't "pickable" which I understood as them not causing a hover effect?) Did you have any particular setting (authors, time interval) enabled?

Reply:
Strange! I'm trying on both Chrome and FF (Mac) in the intersections mode, with all authors enabled and the full time interval. Neither hovering nor clicking is showing me tooltips. Maybe I've misconfigured something?

Reply:
Wait - one thing that just crossed my mind: the tooltip used the built-in DeckGL feature, so that's some plumbing I added to Peripleo. Can you check whether you have the current state of Peripleo checked out from the repository? The plumbing is something I may have added just yesterday, I think.